### ISSUE TITLE
[AVT] The color of hint message could not be seen clearly on FF

### ISSUE THREAD

**Issue by panlei080988** [carbon-design-system/carbon#2036]
## Environment

> Operating system : Win 10

> Browser : Firefox ESR 52.9.0

## Detailed description

> What version of the Carbon Design System are you using?

> What did you expect to happen?

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Create an Internet Service on cloud
2. Click Reliability-> Global Load Balancers item
3. Click on Create pool button

## Additional information

- Screenshots or code
![image](https://media.github.ibm.com/user/25201/files/76bdfe00-36c9-11e9-90cf-d10c7ba7ad08)

**Comment by dakahn** [comment]
Moved the issue as per @vpicone 's suggestion.

**Comment by panlei080988** [comment]
@dakahn Hi, do you know how to verify this on Firefox? I have not seen the fix deployed on to Cloud yet.

**Comment by dakahn** [comment]
@panlei080988 you can test against the deployed preview in the closing PR
https://github.com/IBM/carbon-components/pull/2038#issuecomment-471479488

**Comment by panlei080988** [comment]
@dakahn Do you mean test on https://deploy-preview-2038--the-carbon-components.netlify.com?
If yes, I just want to confirm whether it will reflect on Firefox in future. I have to verify in firefox to close the origin issue.

**Comment by dakahn** [comment]
@panlei080988 yep!
